src/test/regress/expected/set_operations.out

```diff
@@ -0,0 +1,481 @@
+CREATE SCHEMA recursive_union;
```

Comment:
I guess you're still working on the regression tests. But, we should probably add tests for:

* Local tables
* CTEs
* EXCEPT/INTERSECT ALL